hibernate-core/src/main/java/org/hibernate/engine/transaction/jta/platform/internal/JtaPlatformInitiator.java

```diff
@@ -49,6 +49,7 @@ public JtaPlatform initiateService(Map configurationValues, ServiceRegistryImple
 			platform = getFallbackProvider( configurationValues, registry );
 		}
 
+		LOG.usingJtaPlatform( platform != null ? platform.getClass().getName() : "null" );
```

Comment:
IMHO let's not log anything if null.

Reply:
OK, makes sense. Let's wait for the Travis build to pass before merging to master.